math/genvector/inc/Math/GenVector/DisplacementVector3D.h

```diff
@@ -337,18 +337,18 @@ namespace ROOT {
       /**
          return unit vector parallel to this (scalar)
       */
-      template <typename SCALAR = Scalar>
-      typename std::enable_if<std::is_arithmetic<SCALAR>::value, DisplacementVector3D>::type Unit() const
+      template <typename SCALAR = Scalar, typename std::enable_if<std::is_arithmetic<SCALAR>::value>::type * = nullptr>
```

Comment:
(Forget my last comment... having a bad comment day).

You are missing one typename, but yes, I can make that change..